app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,13 @@
+class QuestionsController < ApplicationController
+  def new
+    @question = Question.new
+  end
+
+  def create
+    Question.create!(question_params)
+  end
+
```

Comment:
クラス外から呼び出さないので private メソッドにしておきましょう。

```suggestion
  private
```